client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-intro/styles.scss

```diff
@@ -99,7 +98,7 @@ $heading-font-family: "SF Pro Display", $sans;
 					}
 
 					.formatted-header__subtitle {
-						color: #000;
+						color: var(--studio-gray-60);
```

Comment:
Seen this a couple of times today, lets try and improve the situation in a follow up https://github.com/Automattic/wp-calypso/issues/79819